libs/ui-lib/lib/cim/components/ClusterDeployment/ClusterDetailsFormFields.tsx

```diff
@@ -52,7 +52,7 @@ export const ClusterDetailsFormFields: React.FC<ClusterDetailsFormFieldsProps> =
   cpuArchitectures,
 }) => {
   const { values, setFieldValue } = useFormikContext<ClusterDetailsValues>();
```

Comment:
we should remove `highAvailabilityMode` from `ClusterDetailsValues`
also add the new `controlPlaneAgents` field

Reply:
@rawagner ClusterDetailsValues are shared netween CIM and OCM.
I'll delete the highAvailabilityMode when all works in both sides.
I'll open a task to do in the furute.